stablehlo/dialect/VhloBytecode.cpp

```diff
@@ -321,6 +321,16 @@ enum TypeCode {
   ///   WitnessType {
```

Comment:
Is this PR a good time to update VhloDialect.td to call out the addition of fp8 support to 0.4.0?

Reply:
Yes. Good catch. Will need to rebase on main to avoid conflicts.